nvflare/fuel/hci/client/cli.py

```diff
@@ -122,14 +128,17 @@ def __init__(
             client_key=client_key,
             server_cn=server_cn,
             cmd_modules=modules,
+            overseer_end_point=overseer_end_point,
+            project=project,
+            name=name,
             debug=self.debug,
             poc=poc,
         )
 
         signal.signal(signal.SIGUSR1, partial(self.session_signal_handler))
 
         self.ssid = None
-        self.overseer_agent = self._create_overseer_agent()
+        self.overseer_agent = self._create_overseer_agent(overseer_end_point, project, name)
```

Comment:
When creating the AdminClient, we can already get the overseer-agent instance. No need to create here again.